projects/worker-ccai/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "example-worker-ccai",
+  "author": "Adobe Inc.",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "description": "Asset Compute custom worker leveraging the Content and Commerce Color Extraction API",
+  "repository": "adobe/example-worker-metadata",
+  "dependencies": {
+    "@adobe/asset-compute-sdk": "^2.2.1",
+    "@adobe/asset-compute-xmp": "^1.0.0",
+    "axios": "^0.20.0"
+  },
+  "scripts": {
+    "test": "aio asset-compute test-worker"
```

Comment:
Two change requests:
- add `aio-cli-plugin-asset-compute` as devDependency (`npm install --save-dev @adobe/aio-cli-plugin-asset-compute`)
- change the `test` script to use that by changing it to `adobe-asset-compute test-worker`

Note that `aio app init` with the asset compute template [creates this](https://github.com/adobe/generator-aio-app/blob/master/generators/add-action/asset-compute/index.js#L38). The other example projects here are out of date wrt to this and need to be changed as well (tracked in [NUI-827](https://jira.corp.adobe.com/browse/NUI-827)).

Reply:
Ok, fixed